.github/workflows/send-default-branch-name-to-opslevel.yaml

```diff
@@ -0,0 +1,47 @@
+name: Send default branch name to OpsLevel
+
+on:
+  schedule:
+    - cron: "0 9,12,16 * * *"
+  workflow_dispatch:
+
+concurrency:
+  group: ci-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  get-github-repositories:
+    name: Fetch repositories from GitHub
+    runs-on: ubuntu-latest
+    outputs:
+      repositories: ${{ steps.fetch-all-repositories.outputs.result }}
+
+    steps:
+      - name: Get all repositories
+        uses: actions/github-script@v6
+        id: fetch-all-repositories
+        with:
+          github-token: ${{ secrets.PLEO_READ_TOKEN }}
+          script: |
+            const repositories = await github.paginate("GET /orgs/pleo-io/repos?type=all", {
+              per_page: 100,
+              headers: {
+                "X-GitHub-Api-Version": "2022-11-28",
+              },
+            })
+
+            const mapped = repositories.filter(r => !r.archived).map(r => ({ repository: r.name, defaultBranch: r.default_branch }))
+            return mapped
+
+  send-to-opslevel:
+    name: Send repository data to OpsLevel
+    runs-on: ubuntu-latest
+    needs: get-github-repositories
+    strategy:
+      matrix:
+        repository: ${{ fromJSON(needs.get-github-repositories.outputs.repositories) }}
```

Comment:
Will all of these run at the same time or one at a time?

Reply:
They run roughly 8 at a time